src/Simplex/Chat.hs

```diff
@@ -3162,7 +3160,7 @@ chatCommandP =
       ("/address" <|> "/ad") $> CreateMyAddress,
       ("/delete_address" <|> "/da") $> DeleteMyAddress,
       ("/show_address" <|> "/sa") $> ShowMyAddress,
-      "/auto_accept " *> (AddressAutoAccept <$> onOffP <*> optional (A.space *> msgContentP)),
+      "/auto_accept " *> (AddressAutoAccept <$> onOffP <*> (" incognito=on" $> True <|> " incognito=off" $> False <|> pure False) <*> optional (A.space *> msgContentP)),
```

Comment:
```suggestion
      "/auto_accept " *> (AddressAutoAccept <$> onOffP <*> (" incognito=" *> onOffP <|> pure False) <*> optional (A.space *> msgContentP)),
```

maybe it'll want parens...

Reply:
seems like api may be misused if incognito parameter is not provided and client thinks it's passing "incognito=on" as message.. may be ok to ignore this edge case

Reply:
no, it won't parse like this because of "text" in msgContentP